src/styles/sidebar/components/PaginationNavigation.scss

```diff
@@ -34,17 +36,17 @@
   &__page-button {
     margin: var.$layout-space--xxsmall;
     padding: var.$layout-space--small var.$layout-space;
+
+    &[aria-pressed='true'] {
```

Comment:
I wondered whether we had a convention around using or not-using quotes around attr values. It looks like using single quotes is indeed the convention at present.